cli/src/config.rs

```diff
@@ -98,11 +100,8 @@ impl Into<CoreNodeConfig> for NodeConfig {
             network: self.private_config.network,
             api: self.private_config.api,
             mempool: self.private_config.mempool,
-            services_configs: Default::default(),
-            database: self.private_config.database,
```

Comment:
`database` and `master_key_path` fields of `NodeConfig` were not used by the core code. Is it safe to remove them? Unlike `services_configs`, which was not set by the CLI crate (therefore, I assume it *is* safe to remove), these two fields *are* set by the CLI crate, so the may have a purpose somewhere.

Reply:
`service_configs` is the artifact from pre-DS past. `database`, however, allows (or should allow) tuning RocksDB options and was supported in the past. If it is not used, I'd consider it a bug.

Reply:
As far as I see, all `Node` constructors have DB supplied to them in the assembled form. The `cli` crate itself [does use `database` options from `NodePrivateConfig`](https://github.com/exonum/exonum/blob/3c5bf23c090c6a6cc8ae184ce18f1cab34ac2fc0/cli/src/lib.rs#L173); this code is not changed by this PR. Is this OK?

Reply:
OK, looks like storing the database options in NodeConfig is not needed.